lib/Promise/retry.ts

```diff
@@ -0,0 +1,13 @@
+// tslint:disable-next-line:no-empty
+export async function retry<T>(f: () => Promise<T>, count: number, handleError: (err: any) => void = (err: any) => {}) {
+	for ( let i = 0 ; i < count - 1 ; i++ ) {
```

Comment:
no space between `(` and `let`, also should be `const` not `let`

Reply:
Believe it or not, `const` and `let` have different behavior here -- `let` will be allocated once in the outer scope, while `const` will be allocated repeatedly in the inner. The end result is that, in tight loops, const's memory churn is actually a performance issue.

Because of this, I always use `let` for this sort of loop.

Reply:
Oh, and this project uses tslint instead of eslint which means "auto fix on save" doesn't work :( There will likely be more spacing issues...

Reply:
I was thinking `for (const i of _.range(0, count))`